app/cdap/components/ParsingDrawer/__test__/ParsingDrawer.test.tsx

```diff
@@ -59,7 +54,7 @@ describe('It Should Test the Parsing Drawer Component', () => {
       <Router history={history}>
         <Switch>
           <Route>
-            <ParsingDrawer />
+            <ParsingDrawer setLoading={() => jest.fn()} updateDataTranformation={() => jest.fn()} />
```

Comment:
you should define a function setLoading and use here
so you van asset if the function is being called or not
plz see if this makes sense and talk to me

same for updateDataTranformation

Reply:
assign to unit tester